README.md

```diff
@@ -32,6 +32,7 @@ For other operating systems, please check the [installation guide](https://solan
 After installing the compiler, write the following to flipper.sol:
 
 ```solidity
+@program_id("F1ipperKF9EfD821ZbbYjS319LXYiBmjhzkkf5a26rC") // For Substrate, this line must be removed.
```

Comment:
This is confusing. 

For Solana, you also need a `@payer(..)` tag for the constructor. 

Maybe we should delete the Simple Example

Reply:
@xermicus I'm replying your comment here as well.

I believe we can have a separate MD files for Solana and Substrate. These Solang 101 tutorial is good to have in our main repository instead of the documentation.

Reply:
We have readthedocs documentation already. I think it's better to keep the readme short and to just delete the flipper source code

Reply:
Good idea. The build commands can then use the `flipper.sol` files from the correct directory